### PR TITLE
Create Rule “do-you-do-youtube-livestreams/rule”

### PR DESCRIPTION
**Pull request by StefStar**
Automatically generated by Decap CMS

### CODE REVIEW

**Comment by StefStar** [comment]
1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)
Adam's request for a new rule regarding LiveStreams see email - SSW Events - Live Streaming
2. What was changed?
A new rule created as per Adam's request
3. Did you do pair or mob programming (list names)?
No

**Review by RobJT-wq — APPROVED** [review]
🚀

File: rules/do-you-do-youtube-livestreams/rule.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* Content Repurposing: Livestreams can be recorded and reused as video content, extending their lifespan and reach.
+
+
+
+#### The Downsides of LiveStreaming
+
+* Decline in YouTube Growth: YouTube as a platform is not growing as rapidly as it used to. Instead, viewers are shifting towards other platforms or consuming shorter content like YouTube Shorts. 
+* Reduced In-Person Attendance: LiveStreaming can sometimes reduce the size of the potential in-person audience. Conversely, the convenience of LiveStreaming might lead to procrastination, with people planning to watch online but ultimately skipping it.
+
+#### Tip #1: Don't do watch parties
```
Comment (tiagov8) [inline rules/do-you-do-youtube-livestreams/rule.md:32]: @StefStar 

I would change this to 1 heading "Tips"

And make the 2 list items under it

**Review by tiagov8 — COMMENTED** [review]
cc @RobJT-wq because he was the approver

@StefStar 

- [x] Please action all my comments

File: rules/do-you-do-youtube-livestreams/rule.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+authors:
+  - title: Stef Starcevic
+    url: https://www.ssw.com.au/people/stef-starcevic/
+created: 2024-08-30T11:48:00.000Z
+guid: 8bc8adff-6ed5-465f-80ce-8c2a998f6845
+---
+LiveStreams are a valuable tool for reaching a global audience, yet they come with their own set of challenges. 
+
+They enable us to connect with a worldwide audience in real-time, providing an opportunity to build a larger community over time. In the beginning your LiveStreams may have low attendance, imagine how incredible it could be to have over a thousand attendees. This can only happen if you persistently build your audience month by month. 
+
```
Comment (tiagov8) [inline rules/do-you-do-youtube-livestreams/rule.md:17]: @StefStar 

Your rule is missing the `<!--endintro-->`

- [ ] 1. Please fix

**Comment by tiagov8** [comment]
@StefStar 

FYI - My review was submitted on a merged pull request. (approved by @RobJT-wq)

- [ ] 1. Please make me the reviewer for the next PR ;)

**Pull request merged**